### PR TITLE
Adds methods for API v3

### PR DESCRIPTION
**Pull request by scentless-apprentice**
Adds methods for communicating with v3 of the 3Play API. 
Moves older API methods to `v2` folder.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ install:
   - go install github.com/golangci/golangci-lint/cmd/golangci-lint
 script:
   - golangci-lint run --enable-all -D errcheck -D lll -D dupl -D gochecknoglobals --deadline 5m
-  - go test -v -coverprofile=coverage.txt -covermode=atomic
+  - go test -v ./... -coverprofile=coverage.txt -covermode=atomic
```
Comment (fsouza) [inline .travis.yml]: iirc, Go doesn't like mixing flags and positional args, this should probably fix it:

```suggestion
  - go test -v -coverprofile=coverage.txt -covermode=atomic ./...
```

File: v2/captions.go [hunk 2]
```diff
@@ -1,40 +1,17 @@
-package threeplay
+package v2
 
 import (
 	"errors"
 	"fmt"
+	"github.com/NYTimes/threeplay/common"
```
Comment (fsouza) [inline v2/captions.go]: the latest version of the package is now using ``nytimes`` all lower case. This should fix the build.
```suggestion
	"github.com/nytimes/threeplay/common"
```

File: v2/captions_test.go [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 import (
-	"testing"
-
-	"github.com/nytimes/threeplay"
+	"github.com/nytimes/threeplay/common"
+	"github.com/nytimes/threeplay/v2"
 	"github.com/stretchr/testify/assert"
-	gock "gopkg.in/h2non/gock.v1"
+	"gopkg.in/h2non/gock.v1"
+	"testing"
```
Comment (fsouza) [inline v2/captions_test.go]: Run ``goimports`` to rearrange the imports in 2 separate groups.

File: common/common.go [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package common
```
Comment (fsouza) [inline common/common.go]: Could we have a different name for this package? ``types`` or ``api``, maybe? :thinking:

Comment (scentless-apprentice) [reply common/common.go]: done

File: v2/captions_test.go (hunk 3 above)
Comment (scentless-apprentice) [reply v2/captions_test.go]: I have it on, but it doesn't always work 🤦‍♀

**Pull request merged**